install_pkgs.R

```diff
@@ -0,0 +1,25 @@
+# Create user library and install devtools package
+dir.create(Sys.getenv('R_LIBS_USER'), recursive = TRUE);
+.libPaths(Sys.getenv('R_LIBS_USER'));
+
+# Install devtools and its dependencies
+install.packages('usethis');
+install.packages('covr');
+install.packages('httr');
+install.packages('rversions');
+install.packages('devtools');
+
+# Install packages from requirements list
+requirements_file <- file("r_requirements.txt",open="r")
```

Comment:
Ahh, I see. Maybe I'm confused, but depending on who merges first probably cause undue conflicts. The newer (?) version of this file should probably be used from @sum1lim's PR: https://github.com/pacificclimate/chickadee/blob/i11-wps-ci-netcdf-wrapper/install_pkgs.R

Reply:
Really like this approach, btw :)

Reply:
@nannau that was all Sangwon